packages/env/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Master
+
+### Bug Fixes
+
+- When running scripts using `wp-env run`, the output will not be formatted if not written to terminal display, resolving an issue where piped our redirected output could be unintentionally padded with newlines.
```

Comment:
```suggestion
- When running scripts using `wp-env run`, the output will not be formatted if not written to terminal display, resolving an issue where piping our redirected output could be unintentionally padded with newlines.
```

Reply:
The original was mistaken as you highlight, but it was because it was meant to be "or" (piped or redirected), not "our". This was fixed in 818d22fe5d079c09e5b6f11ec8e95669a7cb1b2b.